gradle/maven.gradle

```diff
@@ -50,6 +51,19 @@ subprojects {
           artifact javadocJar
 
           version project.version
+
+          pom {
+            name = 'Brooklin'
+            description = 'An extensible distributed system for reliable nearline data streaming at scale'
+            url = 'https://github.com/linkedin/brooklin'
```

Comment:
fix the url path.

Reply:
I think this should work just fine, GitHub probably has these redirects configured. But will update anyways.